kolibri/core/assets/src/views/SideNavStorageNotification.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+
+  <div>
+    <div class="sync-status">
+      {{ $tr('deviceStatus') }}
+    </div>
+    <div class="notification">
+      <span>
+        <KIcon
+          icon="error"
+          class="inline-icon"
+          data-test="syncStatusIcon"
+        />
+      </span>
+      <span>
+        <p>{{ $tr('NotEnoughSpace') }}</p>
+      </span>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import { useUserSyncStatus } from '../composables/useUserSyncStatus';
+
+  export default {
+    name: 'SideNavStorageNotification',
+    setup() {
+      const {
+        queued,
+        lastSynced,
+        status,
+        deviceStatus,
+        deviceStatusSentiment,
+      } = useUserSyncStatus();
+      return { queued, lastSynced, status, deviceStatus, deviceStatusSentiment };
+    },
+    props: {},
+    beforeMount() {
+      console.log(this.deviceStatus);
+      console.log(this.lastSynced);
+      console.log(this.status);
+      console.log(this.deviceStatusSentiment);
+      console.log('this is a sidenav notification');
```

Comment:
Not sure if these logs are intentional. I know we do have some console logging in places, so maybe some would be useful here, but some of this might be leftover from debugging